pkg/varlinkapi/system.go

```diff
@@ -90,22 +88,25 @@ func (i *LibpodAPI) GetInfo(call iopodman.VarlinkCall) error {
 		Graph_status:         graphStatus,
 	}
 
-	if len(info) > 2 {
-		registriesInterface := info[2].Data["registries"]
-		if registriesInterface != nil {
-			registries = registriesInterface.([]string)
-		}
-	}
-	if len(info) > 3 {
-		insecureRegistriesInterface := info[3].Data["registries"]
-		if insecureRegistriesInterface != nil {
-			insecureRegistries = insecureRegistriesInterface.([]string)
+	if len(info) > 1 {
+		for key, val := range info[2].Data {
+			if key == "search" {
+				podmanInfo.Registries.Search = val.([]string)
+				continue
+			}
```

Comment:
Why is a search registry treated differently? A search registry could also be marked insecure.

Reply:
libpod/runtime.go sends back two records for registires.

Data type and Search type.  The search type seems to be the old way of sending/storing information in the registries.conf.  

I got this by experimenting around and attempting to make `podman-remote info` look close to `podman info`